content/documentation/maps.md

```diff
@@ -1,25 +1,10 @@
 ---
-title: Zooming Images & Maps
-weight: 4900
+title: Maps
+weight: 4910
 type: essay
-abstract: "Tips and tricks for including interactive maps and images"
+abstract: "Tips and tricks for including basic, interactive maps"
 ---
 
-## Zooming Images
-
-### IIIF & Deep Zooming
-
-IIIF (referred to when speaking as "Triple I F"), stands for the International Image Interoperability Framework. It is a shared set of standards for storing images and image data that allows for seamless and regularized sharing and display of high-resolution images across different uses within an institution, or across different institutions.
-
-This feature adds deep-zooming to any image. You can either point Quire to an existing IIIF repository of images, or add high-resolution images into a Quire project directly.
-
-When adding your own images, Quire processes them into hundreds of individual image tiles. Each tile comprises a portion of the image at a particular zoom level to then be displayed in our existing zooming image viewer. Because the images have been tiled, users of your Quire project only need their browsers to load the tiles of the image portion and zoom level they're looking at at the moment. This means much faster page loads, and higher-resolution images.
-
-{{< q-figure id="iris-zoom" >}}
-
-
-## Maps
-
 <div class="box warning">
 
 - Basic mapping functionality was added into Quire early on but hasn't been used in production on any projects yet, and so remains very rough. Maps are unlikely to be a supported feature moving forward as they can be very demanding and beyond what we can support for the Quire community at large. In the future, we hope to demonstrate some working examples that can act as a guide if you wish to pursue maps yourself. In the meantime, you're welcome to make use of the map functionality as it exists but proceed through this section with caution.
```

Comment:
Is there anything we can do to expand on this section? Or eliminate it and make map info part of the forum?

Reply:
Yeah, it's not great. But my inclination is to leave it as is until we work with Getty Digital on maps and can decide one way or another what (if anything) we should be saying about map functionality. That said, I could be persuaded otherwise.